Library/Formula/libpng.rb

```diff
@@ -28,6 +28,7 @@ class Libpng < Formula
 
   def install
     ENV.universal_binary if build.universal?
+    system "autoreconf", "--force", "--install" if head?
```

Comment:
I normally use the longer names for commands that might not be seen as frequently, but if the short ones are preferred I can make that change.

I wasn't too sure about the head? part because I didn't know where that and/or build was defined. I'll change that